### PR TITLE
Prac 8 code review request

### PR DESCRIPTION
**Pull request by StuBeattie**
Please check formatting, naming and logic.
@CameronOBrien95

### CODE REVIEW

File: CP1404_practicals/prac_08/taxi_simulator.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+MENU = "q)uit, c)hoose taxi, d)rive"
+
+
+def main():
+    """Create a taxi simulator program that utilizes Taxi and SilverServiceTaxi classes.
+    This program will loop through until the user quites."""
+    print("Lets drive!")
+    print(MENU)
```
Comment (CameronOBrien95) [inline CP1404_practicals/prac_08/taxi_simulator.py:20]: You could condense these two lines using string formatting eg. print("Lets drive!\n{}".format(MENU))

File: CP1404_practicals/prac_08/taxi_simulator.py [hunk 2]
```diff
@@ -25,16 +25,34 @@ def main():
...
+                cost = current_taxi.get_fare()
+                print("Your {} trip cost you ${:.2f}".format(current_taxi.name, cost))
+                total_cost += cost
+            else:
+                print("you need to choose a taxi before you can drive")
+        else:
+            print("Invalid option")
+        print("Bill to date: ${:.2f}".format(total_cost))
+        print(MENU)
+        menu_choice = input(">>> ").lower()
```
Comment (CameronOBrien95) [inline CP1404_practicals/prac_08/taxi_simulator.py:53]: You can condense these lines using string formatting

File: CP1404_practicals/prac_08/taxi_simulator.py [hunk 3]
```diff
@@ -25,16 +25,34 @@ def main():
...
+    print("Bill to date: ${:.2f}".format(total_cost))
+    print("Taxis are now:")
+    taxis_available(taxi_types)
```
Comment (CameronOBrien95) [inline CP1404_practicals/prac_08/taxi_simulator.py:56]: you can condense these using string formatting eg. print("Taxis are now:\n{}.format(taxis_available(taxi_types))

**Comment by CameronOBrien95** [comment]
Only things I can falter is string formatting, not much at all, good code. I find if I'm doing 2-3 print() in a row, usually you can condense them to be a little more efficient. GL with the rest of your prac work

File: CP1404_practicals/prac_08/taxi_simulator.py (hunk 1 above)
Comment (StuBeattie) [reply CP1404_practicals/prac_08/taxi_simulator.py:20]: thanks for the advice I will make the changes.

**Pull request merged**